pkg/kube/BUILD.bazel

```diff
@@ -14,3 +14,14 @@ go_library(
         "//vendor/k8s.io/client-go/tools/cache:go_default_library",
     ],
 )
+
+go_test(
```

Comment:
Please add size="small" to tell bazel about the test size. And then please run `make fmt-bazel` to add race="on" to enable race detector and format all build files for consistency.